raiden/network/pathfinding.py

```diff
@@ -28,13 +28,13 @@ def get_pfs_info(url: str) -> typing.Optional[typing.Dict]:
         return None
 
 
-def get_random_service(service_registry: ServiceRegistry) -> Optional[str]:
+def get_random_service(service_registry: ServiceRegistry) -> Tuple[Optional[str], Optional[str]]:
```

Comment:
Since you are changing the interface of this function and it now returns a tuple can you add a docstring on howthis function is now supposed to function?